src/Focusable.ts

```diff
@@ -129,37 +117,35 @@ export class FocusableAPI implements Types.FocusableAPI {
         const attrVal = el.getAttribute("aria-hidden");
 
         if (attrVal && attrVal.toLowerCase() === "true") {
-            return true;
+            if (!this._tabster.modalizer?.isAugmented(el)) {
+                return true;
+            }
         }
 
         return false;
     }
 
     findFirst(options: Types.FindFirstProps): HTMLElement | null | undefined {
         return this.findElement({
```

Comment:
maybe you can now just pass options - valid for each of these 3 findElement calls

Reply:
Not sure what you mean.